src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+@import 'variables';
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 $header-padding;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo-img {
+    display: flex;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: $header-margin;
```

Comment:
The variable `$header-margin` should be defined in the 'variables' file or elsewhere before it is used here. Ensure that it is correctly imported or defined.